test/cpp/qps/qps_json_driver.cc

```diff
@@ -86,6 +86,7 @@ static std::unique_ptr<ScenarioResult> RunAndReport(const Scenario& scenario,
   result->mutable_scenario()->CopyFrom(scenario);
 
   GetReporter()->ReportQPS(*result);
+  GetReporter()->ReportAttemptedQPS(*result);
```

Comment:
This is ~~chicken nuggets~~ a constant, right? Why report something that's just part of the client config which is the input to the scenario?

Reply:
It doesn't need to be logged I guess, but it does need to exist in the ReportSummary proto. This is to match internal behavior for the Stubby benchmarking system.

Both sent the report protos to a Result Server, which is responsible for writing the data somewhere where perfkit can read it (for internal dashboards).

Ultimate goal is to be able to dashboard open loop scenarios